src/roles/nuh-deploy/templates/config.yml.j2

```diff
@@ -40,22 +40,35 @@ network:
 {% endfor %}
 {% endif %}
     nuagenetwork: Internal
+{% if vrrp is defined %}
+vrrp:
+{% for vrrp_item in vrrp %}
```

Comment:
What if there are no entries? (empty list)

Reply:
Added checks for not empty and tested